source/ubuntu.rst

```diff
@@ -0,0 +1,57 @@
+Ubuntu 20 04 on Amazon EC2 
+==========================
+Use Ubuntu 20.04, on Amazon Elastic Compute Cloud (Amazon EC2). 
+We have greatly simplified the parameters that you have to enter to create an instance so that you can get started very quickly. The product launches in a matter of a few minutes and is ready to go.
+
+Parameters
+-----------
+
+.. list-table:: 
+   :widths: 50, 50
+   :header-rows: 1
+
+   * - Parameter
+     - Details
+   * - Product Name
+     - Provide a name to help you easily identify this instance of the product. Only alphanumeric characters, dots, hyphens and underscores are allowed. Spaces and special characters are not allowed. Eg: MedicalResearch
+   * - KeyPair
+     - Choose a KeyPair in the dropdown list. Note: If KeyPair is not available in the drop-down, click on the “+” button. A KeyPair creation form is opened. Fill the details in the form and click on the “Create KeyPair” button. Now that KeyPair is available in the list. Remember to save the private key file securely for future use. Do not share this file with others for the security of your account.
+   * - LatestAmiId
+     - LatestAmiId - Provide the location from where to pick the latest AMI on which the instance should be based. Valid values are: Amazon Linux 2 - /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
```

Comment:
The value provided is for Amazon Linux 2. The valid value for Ubuntu image should be picked from the Ubuntu yaml file.